content/blog/2021/cfp-announcement.md

```diff
@@ -0,0 +1,100 @@
+Title: Call for Proposals for PyCon India 2021 is now open!
+Date: 2021-03-04 17:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2021
+Slug: cfp-announcement
+Summary: PyCon India is back for 2021, it's online and the [call for proposals](https://in.pycon.org/cfp/2021/proposals/) has been officially opened!
+
+2020 was a rollercoaster for all of us. The PyCon India team wishes you a safe and prosperous year. After accessing the circumstances, we have decided to organize PyCon India 2021 online. You can know more about that from our [Announcement blog](https://in.pycon.org/blog/2021/pycon-india-announcement.html).
+
+So, whether you are someone submitting a proposal for talks for the first time or just want to share what you have been up to with Python for a long time. This is YOUR time to shine!
+
+We are looking forward to seeing beginner, intermediate, and advanced proposals on topics mentioned below from speakers from a variety of experience levels and fields.  You don’t need to be a veteran who has spoken at dozens of conferences. For making our conference truly diverse, we are looking for all kinds of people. That’s what our community is comprised of, and that’s what this conference’s schedule should be made from.
+
+Please make note of some important dates as mentioned below:
+
+- CFP opens on **4th March 2021**

```

Comment:
Need to update this date since it has been pushed forward by a couple of days.

Reply:
Done